catalog/src/main/java/com/qubole/quark/catalog/db/SchemaFactory.java

```diff
@@ -78,21 +76,19 @@
    */
   public QuarkFactoryResult create(Properties info) throws QuarkException {
     try {
-      ObjectMapper objectMapper = new ObjectMapper();
-      objectMapper.registerModule(new GuavaModule());
-      DbCredentials dbCredentials = objectMapper
-          .readValue((String) info.getProperty("dbCredentials"), DbCredentials.class);
```

Comment:
Since we are not using DbCredentials class in catalog module anymore, can we delete it?